npm-shrinkwrap.json

```diff
@@ -369,8 +369,8 @@
     },
     "fxa-auth-db-mysql": {
       "version": "0.59.0",
-      "from": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#a0522c1eda66f9fa92073f4f99959af78dec3490",
-      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#a0522c1eda66f9fa92073f4f99959af78dec3490",
+      "from": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#7aa75f4cc106b9040063e3a2d1a1b8ddefe8e87a",
```

Comment:
Can you update this to `afcfe858d17ade910e3aa0df3064976d5942223e` to get @philbooth's latest changes?